mklove/modules/configure.libzstd

```diff
@@ -6,15 +6,18 @@
 #   mkl_require libzstd
 #
 # And then call the following function from the correct place/order in checks:
-#   mkl_check libzstd [<action>]
+#   mkl_check libzstd
 #
 
-mkl_toggle_option "Feature" ENABLE_ZSTD "--enable-zstd" "Enable support for ZSTD compression" "y"
+mkl_toggle_option "Feature" ENABLE_ZSTD "--enable-zstd" "Enable support for ZSTD compression" "d"
```

Comment:
Also need to make sure this still works with automatic dependency installation, e.g, `./configure --install-deps` and `./configure --install-deps --source-deps-only`.

Reply:
Yep, `--install-deps` and `--install-deps` --source-deps-only` both are unaffected by this change!